NuKeeper.AzureDevOps/AzureDevopsPlatform.cs

```diff
@@ -88,6 +94,48 @@ public async Task OpenPullRequest(ForkData target, PullRequestRequest request, I
                 }
             };
 
+            // todo: maybe they don't want to create reviewers during the pull request itself, but add individual reviewers later on?
+            // could be ok, if you want to make sure the pull request creation doesn't fail, but you can simply continue without reviewers
+            // anyway. I don't see any advantage to adding reviewers/labels separately.
+
+
+            // It seems there's an api to search for a user based on email address, maybe we should just use that?
+            // but I guess in theory the user should be part of the team...?
+            if (request.Reviewers.Any())
```

Comment:
Think we can remove this comments and documented the strategy that was chosen and why

Reply:
Done.